src/config/configStore.ts

```diff
@@ -417,7 +374,7 @@ export abstract class BaseConfigStore<
     return this.crypto.isEncrypted(value) ? value : this.crypto.encrypt(value);
   }
 
-  protected decrypt(value: unknown): Optional<string> {
+  protected decrypt(value: unknown): string | undefined {
```

Comment:
Are we starting to move away from `ts-types`? 🏆

Reply:
I don't find this one useful (saving 2 characters, but an extra shift-strike?), for the low cost of having to think, "now does that do nulls or not?  or wait, no, that Nullable which also allows both undefined and null"

There were several places in this PR that I **really** needed `entriesOf` because native TS is overly conservative about object props.